src/test/token/ERC20MultiVotes.t.sol

```diff
@@ -77,6 +77,14 @@ contract ERC20MultiVotesTest is DSTestPlus {
         }
     }
 
+    function testDelegateToAddressZero() public {
```

Comment:
For tests which I'm expecting to validate a sad path of the code, I normally name to something like `testCanNotDelegateToAddressZero()`. On reading this test name, I'd have expected that it would test that you could delegate to address zero. Minor though

Reply:
Renamed some tests!